src/components/transactions/eip5792/sendCalls.js

```diff
@@ -0,0 +1,254 @@
+import globalContext from '../../..';
+
+const DEFAULT_CALLS = [
+  {
+    to: '0x0c54FcCd2e384b4BB6f2E405Bf5Cbc15a017AaFb',
+    data: '0x654365436543',
+    value: '0x1234123412341234',
```

Comment:
mmm that's `4.837 ETH`, for the e2e is fine, but in the case we want to manually use the test dapp that's a ton of money :laughing: could we maybe tweak the values to be either a very low amount? Enough for testing simulations, but still very few to be able to perform a couple of manual flows

![Screenshot from 2025-02-11 17-15-19](https://github.com/user-attachments/assets/4d496fc7-2574-4a37-ab42-5e3f6c72f012)

Reply:
Good point, will lower the default, but these also can be customised via `Edit Request`.

Reply:
oh yeahhh, make sense, thank you!